jolt-sdk/src/host_utils.rs

```diff
@@ -17,37 +10,7 @@ pub use jolt_core::host;
 pub use jolt_core::jolt::instruction;
 pub use jolt_core::jolt::vm::{
     bytecode::BytecodeRow,
-    rv32i_vm::{RV32IJoltProof, RV32IJoltVM, RV32I},
+    rv32i_vm::{RV32IHyraxProof, RV32IJoltProof, RV32IJoltVM, PCS, RV32I},
     Jolt, JoltCommitments, JoltPreprocessing, JoltProof,
 };
 pub use tracer;
-
-pub type CommitmentScheme = HyraxScheme<G>;
-
-#[derive(CanonicalSerialize, CanonicalDeserialize)]
-pub struct Proof {
-    pub proof: RV32IJoltProof<F, CommitmentScheme>,
-    pub commitments: JoltCommitments<CommitmentScheme>,
-}
```

Comment:
can we leave this where it is? You can enable the `host` feature of this package to make it `std`

Reply:
I moved this due to issues with compiling Jolt behind rust -> go ffi bindings with the host feature enabled.